pints/_optimisers/__init__.py

```diff
@@ -201,6 +204,18 @@ def _suggested_population_size(self):
         """
         raise NotImplementedError
 
+    def n_hyper_parameters(self):
+        """ See :meth:`TunableMethod.n_hyper_parameters()`. """
+        return 1
+
+    def set_hyper_parameters(self, x):
+        """
+        Hyper-parameter vector is [population_size]
```

Comment:
Could you update all these to the form "The hyper-parameter vector is ``[lalala]``." ?

Reply:
Sure, can do.